fastf1/api.py

```diff
@@ -193,7 +193,7 @@ def requests_post(cls, *args, **kwargs):
         return cls._requests_session.post(*args, **kwargs)
 
     @classmethod
-    def clear_cache(cls, cache_dir, deep=False):
+    def clear_cache(cls, cache_dir=None, deep=False):
         """Clear all cached data.
 
         This deletes all cache files in the provided cache directory.
```

Comment:
"If no cache directory is specified, the default cache directory is assumed", or something along those lines. Else it is completely unclear, what happens if you don't provide a value for `cache_dir`.

Reply:
We should write `will delete a random folder on your computer unless a specific directory is provided` to see if anyone reads the docs :grin: 

But anyhow... Fixed